components/general-timetables-list.vue

```diff
@@ -11,7 +11,8 @@
       :key="path"
       no-action>
       <v-list-tile slot="activator">
-        <v-list-tile-content>
+        <v-list-tile-content
+          @click="trackMatomoEvent('Menu','normal plan used', path)">
```

Comment:
path ist `Informatik Bachelor of Science`, das hier macht die erste Ebene auf. Z. 38 und Z. 49 sind die Nuxt-Links. Track dann den Wert genau wie bei den Favoriten, damit man direkt vergleichen kann 😃